meta-lmp-base/recipes-bsp/u-boot/u-boot-tools_%.bbappend

```diff
@@ -0,0 +1,3 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI_append = "file://0001-tools-image-host-fix-wrong-return-value.patch"
```

Comment:
Either add an extra space before file (when using append) or simply just use SRC_URI += "file..." here.

Append does a direct string append, and it is safer to use it with an extra space.

Reply:
fixed, thanks!